ios/engine/KMEI/KeymanEngine/Classes/Keyboard/KeymanWebViewController.swift

```diff
@@ -425,11 +425,11 @@ extension KeymanWebViewController {
     }
   }
 
-  func showBanner(_ display: Bool) {
-    let message = "Changing banner's alwaysShow property to \(display)"
+  func setUpBanner() {
+    let message = "Changing banner's alwaysShow property to true"
     os_log("%{public}s", log: KeymanEngineLogger.settings, type: .debug, message)
     SentryManager.breadcrumb(message, category: "engine", sentryLevel: .debug)
-    webView?.evaluateJavaScript("showBanner(\(display ? "true" : "false"))", completionHandler: nil)
+    webView?.evaluateJavaScript("showBanner(true)", completionHandler: nil)
```

Comment:
This looks like we could eliminate `showBanner()` altogether in ios-host.js and fold it into the setup functionality, and then we can remove this function too.